src/components/ItemPriceRange/ItemPriceRange.tsx

```diff
@@ -45,6 +46,13 @@ export function ItemPriceRange(props: Props) {
     let history = useHistory();
     let [selectedDateRange, setSelectedDateRange] = useState(DEFAULT_DATE_RANGE);
 
+    if(props.disableAllTime && selectedDateRange === DateRange.ALL){
+        setSelectedDateRange(DateRange.MONTH);
+        if (props.onRangeChange) {
+            props.onRangeChange(getTimeSpanFromDateRange(DateRange.MONTH));
+        }
+    }
+
```

Comment:
This should not have changed in this PullRequest.
All changes in this file are part of #116 ad should be reversed in this branch